test/d2l-applied-filters/d2l-applied-filters.html

```diff
@@ -3,15 +3,15 @@
 	<head>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
-
-		<title>d2l-filter-dropdown test</title>
-
-		<script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-		<script src="../../../wct-browser-legacy/browser.js"></script>
-		<script type="module" src="../../components/d2l-filter-dropdown/d2l-filter-dropdown.js"></script>
-		<script type="module" src="../../components/d2l-filter-dropdown/d2l-filter-dropdown-category.js"></script>
-		<script type="module" src="../../components/d2l-filter-dropdown/d2l-filter-dropdown-option.js"></script>
-		<script type="module" src="../../components/d2l-applied-filters/d2l-applied-filters.js"></script>
+		<title>d2l-applied-filters test</title>
+		<script src="/node_module/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
```

Comment:
typo: `/node_modules/`

Reply:
Ah, thanks lol.  Weirdly enough, this did not fail tests locally